src/runtime-rs/crates/hypervisor/src/device/device_manager.rs

```diff
@@ -25,19 +25,22 @@ pub type ArcMutexDevice = Arc<Mutex<dyn Device>>;
 /// block_index and released_block_index are used to search an available block index
 /// in Sandbox.
 ///
-/// @block_index generally default is 1 for <vdb>;
+/// @block_index generally default is 0 for <vda>;
 /// @released_block_index for blk devices removed and indexes will released at the same time.
+/// @pmem_index is to indicate the index of the pmem device
 #[derive(Clone, Debug, Default)]
 struct SharedInfo {
     block_index: u64,
     released_block_index: Vec<u64>,
+    pmem_index: u64,
```

Comment:
We need carefully  to consider whether to add a `pmem_index` field like this.  And, if not so, how to handle the block_index to satisfy the pmem index requirement.